src/main/groovy/org/octopusden/octopus/license/management/plugins/gradle/LicenseGradlePlugin.groovy

```diff
@@ -121,8 +122,9 @@ class LicenseGradlePlugin implements Plugin<Project> {
         Task processLicenses = project.getTasks().create(processLicensesTaskName, LicenseTask.class)
         processLicenses.dependsOn(processLicensedDependencies)
         processLicenses.onlyIf {
-            "false".equalsIgnoreCase(project.findProperty("license.skip") as String ?: "true") ||
-                    "false".equalsIgnoreCase(MavenParametersUtils.getProjectProperty(project, "license.skip") ?: "true")
+            def licenseSkipPropertyValue = MavenParametersUtils.getProjectProperty(project, "license.skip")
+                    ?: project.findProperty("license.skip")
+            return "false".equalsIgnoreCase(licenseSkipPropertyValue as String)
```

Comment:
![image](https://github.com/user-attachments/assets/877f4511-bfc1-4275-ac0d-e6575a39b715)
in last case, we need run plugin

Reply:
Okay, I've updated the code. I also handle cases where the value is `"null"`, which can occur when retrieved from `maven-license-parameters`.